repos/system_upgrade/common/actors/systemfacts/actor.py

```diff
@@ -41,3 +55,5 @@ def process(self):
         self.produce(systemfacts.get_selinux_status())
         self.produce(systemfacts.get_firewalls_status())
         self.produce(systemfacts.get_firmware())
+        self.produce(systemfacts.get_default_grub())
```

Comment:
what about `get_default_grub_configuration`? or `get_default_grub_conf`? As well, we should check the file exists. On some systems upgraded from RHEL 6 the file does not have to be present in case that users have not finished the migration to GRUB 2. we should check it to be able to give people proper hints what's wrong in such a case.